packages/indexer-agent/src/agent.ts

```diff
@@ -599,6 +622,27 @@ class Agent {
     })
   }
 
+  async reconcileCostModel(
+    costModels: CostModelAttributes[],
+    targetAllocations: SubgraphDeploymentID[],
+  ): Promise<void> {
+    const costModelsIDs = costModels
+      .filter(model => model.deployment != COST_MODEL_GLOBAL)
+      .map(model => new SubgraphDeploymentID(model.deployment))
+    const deploymentMissingModels = targetAllocations.filter(
+      allocation => !(allocation.bytes32 in costModelsIDs),
+    )
+
+    if (deploymentMissingModels) {
+      this.logger.warn(
+        `The following deployments does not have defined cost models and uses global if it exists`,
```

Comment:
Since using plural `deployments` here it should be `do not`, so: 
```
The following deployments do not have defined cost models and will use the global cost model if it exists
```